scrapy_zyte_api/providers.py

```diff
@@ -146,6 +166,33 @@ async def __call__(  # noqa: C901
             results.append(response)
             self.update_cache(request, {BrowserResponse: response})
 
+        if AnyResponse in to_provide:
+            if "browserHtml" in api_response.raw_api_response:
+                any_response = AnyResponse(
+                    response=BrowserResponse(
```

Comment:
A small thing, but probably it'd be better to reuse BrowserResponse instance if it's already created. This way we e.g. won't parse HTML twice.

Reply:
Good call. Added a test for it as well https://github.com/scrapy-plugins/scrapy-zyte-api/pull/161/commits/5c00cdc76c50988a21ff1c5cd1fc83a69242c204#diff-864023bf5f9f833678b86d78cf8354385edaf4439d52876ab6b819bfe58d8c35R323-R328